tests/jballerina-unit-test/src/test/resources/test-src/annotations/annots_with_invalid_type.bal

```diff
@@ -20,3 +20,13 @@ type VAL true;
 type trueArray VAL[];
 
 annotation trueArray v2 on type;
+
+annotation record {| Foo f; |} v3 on type;
```

Comment:
I personally prefer the current way (We use this style in nBallerina too). However, our formatter would format like this though. 
```suggestion
annotation record {|Foo f;|} v3 on type;
```

Reply:
Shall we leave as is (the string representation also uses spaces)? 

I think it helps readbility for inclusive/exclusiveness too

```
annotation record {lName name;} a on type;
// vs
annotation record {|lName name;|} b on type;
```

vs

```
annotation record { lName name; } a on type;
// vs
annotation record {| lName name; |} b on type;
```

I'm not sure why we don't have spaces here, may have taken a similar approach to mapping constructors, but they aren't comparable IMO.